org_status/org_status.py

```diff
@@ -75,10 +76,25 @@ def get_argument_parser():
     parser.add_argument('--verbose', '-v', action='store_true')
     parser.add_argument('--hosts-only', '-o', action='store_true')
     parser.add_argument('--skip-host-checks', action='store_true')
+    parser.add_argument('--export-orgs', action='store_true')
+    parser.add_argument('--format', type=str)
 
     return parser
 
 
+def export(repo_data, exporter_name):
+    exporters = get_all_supported_exporters()
+
+    for exporter in exporters:
+        if exporter.NAME == exporter_name:
+            try:
+                exporter.convert_org_list_to_format(repo_data)
+            except NotImplementedError:
+                print('exporter does not support exporting results')
```

Comment:
Please include the name of the encoder in this message, and should probably be `styled` red.